src/registrar/templates/application_sidebar.html

```diff
@@ -10,11 +10,14 @@
           {{ form_titles|get_item:this_step }}
         </a>
       {% else %}
-        <li class="usa-sidenav__item step--locked">
-        {{ form_titles|get_item:this_step }}
-        <svg class="usa-icon" aria-labelledby="locked-step" role="img">
-          <title id="locked-step__{{forloop.counter}}">locked until previous steps have been completed </title>
-          <use xlink:href="{%static 'img/sprite.svg'%}#lock"></use>
+        <li class="usa-sidenav__item sidenav__step--locked">
+          <span>
+            <svg class="usa-icon" aria-labelledby="locked-step" role="img">
+              <title id="locked-step__{{forloop.counter}}">locked until previous steps have been completed </title>
+              <use xlink:href="{%static 'img/sprite.svg'%}#lock"></use>
+            </svg>
+              {{ form_titles|get_item:this_step }}
```

Comment:
```suggestion
              <title id="locked-step__{{forloop.counter}}">lock icon</title>
              <use xlink:href="{%static 'img/sprite.svg'%}#lock"></use>
            </svg>
              {{ form_titles|get_item:this_step }}
              <span class="sr-only">, locked until previous steps have been completed</span>
```

**Suggestion:** With the icons on the left side of the nav (which looks much nicer, btw), the text ordering has become a little odd. It reads:

"locked until previous steps have been completed.  Organization name and mailing address"

Consider shorten the title and moving the assistive text to follow the form title.